engine/src/cmd/basecomputer.cpp

```diff
@@ -284,7 +284,8 @@ static const ModeInfo modeInfo[] = {
         ModeInfo("Missions BBS  ", "Missions", "MissionsMode", "MissionsGroup"),
         ModeInfo("GNN News  ", "News", "NewsMode", "NewsGroup"),
         ModeInfo("Info/Stats  ", "Info", "InfoMode", "InfoGroup"),
-        ModeInfo("Load / Save ", "LoadSave", "LoadSaveMode", "LoadSaveGroup")
+        ModeInfo("Load / Save ", "LoadSave", "LoadSaveMode", "LoadSaveGroup"),
```

Comment:
I'm going to assume you simply reverted my deletions here.
I'm not a big fan of this. 
It shouldn't be too hard to figure out how to prevent a crash without all this jazz.

Why don't I take a stab at it tomorrow and you separate it out of the PR?